demos/tcp/aws_tcp_echo_client_single_task.c

```diff
@@ -402,6 +425,12 @@ static void prvEchoClientTask( void * pvParameters )
          * congested. */
         vTaskDelay( echoLOOP_DELAY );
     }
+
+    /* Notify the parent task about completion. */
+    xSemaphoreGive( EchoSingleSemaphore );
+
+    /* Allow the task to delete itself. */
```

Comment:
I am not sure about this comment, I think it should say "Delete self" not "Allow to delete"

Reply:
Done, thanks!